src/TrackPanel.cpp

```diff
@@ -870,6 +870,52 @@ std::shared_ptr< TrackPanelCell > TrackPanel::GetBackgroundCell()
    return mpBackground;
 }
 
+namespace {
+std::vector<int> FindAdjustedChannelHeights( Track &t )
```

Comment:
Well, the reason why constants aren't needed outside the TrackPanel.cpp is the fact that they are all accomodated within the 'height' of the TrackView? If so, then meaning of that height will depend on the context, and example of such is printing. Printing context isn't aware of other decorative elements and thus printed page look different.

Reply:
That is true.  And that was already a problem just for printing.  I mentioned that printing already used the same track height values but did not make the same track and channel separators.

There are also differences in how printing represents the timeline.

There was already a mismatch between the print and the screen appearance.  Let the mismatch remain.  It's not in scope to fix it.  If it becomes important to fix it, then we can make TrackPanel responsible for the printed image too.